pages/about.html

```diff
@@ -4,9 +4,44 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="/styles/about.css" rel="stylesheet">
 </head>
 <body>
-  
+  <header> 
+        <nav>
+          <a href="/index.html">Home</a>
+          <a href="/pages/portfolio.html">Portfolio</a>
+          <a href="/pages/about.html">About</a>
+          </nav>
+      </header> 
```

Comment:
Thanks for adding in navigation to all the pages to aviod dead ends.

Reply:
If we wanted to avoid absolute paths here, they would be written as
```html
          <a href="../index.html">Home</a>
          <a href="portfolio.html">Portfolio</a>
          <a href="about.html">About</a>
```

`index.html` is one folder up (`../`), while the other pages are here in the same folder (so no directory needed in the path).